lib/inspec/rspec_json_formatter.rb

```diff
@@ -105,50 +106,36 @@ def initialize(*args)
     @backend = nil
   end
 
+  attr_reader :profiles
```

Comment:
You're right it doesn't need to be public. The methods associated with the interface for formatter and `#add_profile` are likely the only things that need to be public. Will definitely make the change and make more things `private`.